lang/es.js

```diff
@@ -67,6 +67,15 @@ var language = {
       'title':'Envíanos tus comentarios sobre el editor de Python',
       'label':'Enviar Comentarios'
     },
+    'en': {
+      'title': 'Inglés'
+    },
+    'es': {
+      'title': 'Español'
+    },
+    'pl': {
+      'title': 'Polaco'
+    },
```

Comment:
About the translations, have you checked with @microbit-mark how to do this via crowdin?